examples/get_events/main.go

```diff
@@ -38,23 +37,16 @@ func main() {
 
 func demo(deployedContract *flow.Account, runScriptTx *flow.Transaction) {
 	ctx := context.Background()
-	flowClient := examples.NewFlowClient()
+	flowClient, err := http.NewClient(http.EmulatorHost)
+	examples.Handle(err)
 
 	// Query for account creation events by type
-	result, err := flowClient.GetEventsForHeightRange(ctx, client.EventRangeQuery{
-		Type:        "flow.AccountCreated",
-		StartHeight: 0,
-		EndHeight:   100,
-	})
+	result, err := flowClient.GetEventsForHeightRange(ctx, "flow.AccountCreated", 0, 30)
```

Comment:
this used to be 0 to 100. Is 30 enough?

Reply:
Not sure what was reasoning for this to be changed, but in any case, I don't believe it needs to be 100 or even 30, but I will check and make sure.